arithmetization/src/main/java/net/consensys/linea/zktracer/module/trm/Trm.java

```diff
@@ -55,36 +50,10 @@ public Address callTrimming(Bytes32 rawHash) {
   }
 
   public Address callTrimming(Bytes addressToTrim) {
-    Bytes32 addressPadded = Bytes32.leftPad(addressToTrim);
+    final Bytes32 addressPadded = Bytes32.leftPad(addressToTrim);
     return callTrimming(addressPadded);
   }
 
-  @Override
-  public void traceStartTx(WorldView world, TransactionProcessingMetadata txMetaData) {
```

Comment:
This was a duplicate, as in the warming macro section we `makeWithTrm` the account fragment (which is the right way to do.